lib/literal/array.rb

```diff
@@ -196,6 +205,27 @@ def reject!(...)
 		self
 	end
 
+	def replace(value)
+		case value
+		when Array
```

Comment:
Does this need to be `::Array`? I’m not sure if it’s going to resolve to itself here since the class is called `Array`.